net/freeradius3/patches/004-get-hostname-from-variable-in-radtest.patch

```diff
@@ -0,0 +1,12 @@
+--- a/src/main/radtest.in
++++ b/src/main/radtest.in
+@@ -112,7 +112,7 @@ if [ "$7" ]
+ then
+ 	nas=$7
+ else
+-	nas=`hostname`
++	nas=$HOSTNAME
```

Comment:
I think $HOSTNAME is bash specific. You could do 

```
$(cat /proc/sys/kernel/hostname)
```

instead.

Reply:
Hi, Rosen.
Thanks for your review.
Indeed, this variable is configured in busybox by BASH_HOSTNAME_VAR.
Corrected as you suggested.